Add ModelTest app for lesson 4 in Stream3 solutions

NielMc opened the pull request:
No solution for this lesson currently on GitHub.

NielMc committed:
Add ModelTest app for lesson 4

pep8speaks commented:
Hello @NielMc! Thanks for updating the PR.

 - There are no PEP8 issues in the file [`Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_app/admin.py`](https://github.com/NielMc/full_stack_solutions/blob/0fef6b6709e9a19052ef48e1439903e546cafddf/Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_app/admin.py) !

 - There are no PEP8 issues in the file [`Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_app/apps.py`](https://github.com/NielMc/full_stack_solutions/blob/0fef6b6709e9a19052ef48e1439903e546cafddf/Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_app/apps.py) !

 - There are no PEP8 issues in the file [`Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_app/models.py`](https://github.com/NielMc/full_stack_solutions/blob/0fef6b6709e9a19052ef48e1439903e546cafddf/Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_app/models.py) !

 - There are no PEP8 issues in the file [`Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_app/tests.py`](https://github.com/NielMc/full_stack_solutions/blob/0fef6b6709e9a19052ef48e1439903e546cafddf/Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_app/tests.py) !

 - There are no PEP8 issues in the file [`Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_app/views.py`](https://github.com/NielMc/full_stack_solutions/blob/0fef6b6709e9a19052ef48e1439903e546cafddf/Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_app/views.py) !

 - In the file [`Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_prj/settings.py`](https://github.com/NielMc/full_stack_solutions/blob/0fef6b6709e9a19052ef48e1439903e546cafddf/Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_prj/settings.py), following are the PEP8 issues :

> [Line 87:80](https://github.com/NielMc/full_stack_solutions/blob/0fef6b6709e9a19052ef48e1439903e546cafddf/Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_prj/settings.py#L87): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (91 > 79 characters)
> [Line 90:80](https://github.com/NielMc/full_stack_solutions/blob/0fef6b6709e9a19052ef48e1439903e546cafddf/Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_prj/settings.py#L90): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (81 > 79 characters)
> [Line 93:80](https://github.com/NielMc/full_stack_solutions/blob/0fef6b6709e9a19052ef48e1439903e546cafddf/Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_prj/settings.py#L93): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (82 > 79 characters)
> [Line 96:80](https://github.com/NielMc/full_stack_solutions/blob/0fef6b6709e9a19052ef48e1439903e546cafddf/Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_prj/settings.py#L96): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (83 > 79 characters)

 - There are no PEP8 issues in the file [`Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_prj/urls.py`](https://github.com/NielMc/full_stack_solutions/blob/0fef6b6709e9a19052ef48e1439903e546cafddf/Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_prj/urls.py) !

 - There are no PEP8 issues in the file [`Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_prj/wsgi.py`](https://github.com/NielMc/full_stack_solutions/blob/0fef6b6709e9a19052ef48e1439903e546cafddf/Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_prj/wsgi.py) !

 - There are no PEP8 issues in the file [`Stream-Three/Unit04-Hello_Django_Administration/ModelTest/manage.py`](https://github.com/NielMc/full_stack_solutions/blob/0fef6b6709e9a19052ef48e1439903e546cafddf/Stream-Three/Unit04-Hello_Django_Administration/ModelTest/manage.py) !



##### Comment last updated on March 08, 2017 at 16:48 Hours UTC

aaronsnig501 commented:
Hey @NielMc, there's a stray comma here -https://github.com/NielMc/full_stack_solutions/blob/371781a6c891ed14faf27b4978f8c594d703d72b/Stream-Three/Unit04-Hello_Django_Administration/ModelTest/ModelTest_prj/settings.py#L58

Could you resolve that and then I'll merge it in :)

NielMc committed:
remove trailing comma in settings.py in ModelTest app for lesson 4

NielMc committed:
Moved template directory and settings.py L58 spacing from ModelTest app for lesson 4

NielMc commented:
Removed that line 58 extra carriage return and moved templates directory to
app level as per LMS.

On 8 Mar 2017 4:35 p.m., "Aaron Sinnott" <notifications@github.com> wrote:

> Hey @NielMc <https://github.com/NielMc>, there's a stray comma here -
> https://github.com/NielMc/full_stack_solutions/blob/
> 371781a6c891ed14faf27b4978f8c594d703d72b/Stream-Three/Unit04-Hello_Django_
> Administration/ModelTest/ModelTest_prj/settings.py#L58
>
> Could you resolve that and then I'll merge it in :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Code-Institute-Org/full_stack_solutions/pull/10#issuecomment-285092771>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ASfgRvqxSmb3fqBiXU6LPlTQYvCm90s_ks5rjtjfgaJpZM4MW8zg>
> .
>
